tests/test_local_python_executor.py

```diff
@@ -41,6 +40,21 @@
 )
 
 
+# Non-exhaustive list of dangerous modules that should not be imported
+DANGEROUS_MODULES = [
+    "builtins",
+    "io",
+    "multiprocessing",
+    "os",
+    "pathlib",
+    "pty",
+    "shutil",
+    "socket",
+    "subprocess",
+    "sys",
+]
+
+
```

Comment:
```suggestion
```
I guess this is no-longer needed?

Reply:
Just here for test coverage: whichever modification of the code, we want these modules to be forbidden.